cloudbuild.yaml

```diff
@@ -14,6 +14,6 @@ steps:
         - name: gcr.io/cloud-builders/docker
           args:
                 - 'build'
-                - '--tag=%REPO%/debian:%TAG%'
+                - '--tag=%REPO%/debian%NUM%:%TAG%'
```

Comment:
We're getting a few of these now, would you mind adding some comments to the top of this yaml file explaining what each of the variables are?